packages/kit/src/exports/public.d.ts

```diff
@@ -346,13 +346,7 @@ export interface KitConfig {
 	/**
 	 * Here be dragons. Enable at your peril.
 	 */
-	dangerZone?: {
-		/**
-		 * Automatically add server-side `fetch`ed URLs to the `dependencies` map of `load` functions. This will expose secrets
-		 * to the client if your URL contains them.
-		 */
-		trackServerFetches?: boolean;
```

Comment:
hmm. I wonder if we should leave this in the docs for now with a note that it was only available in 1.0 in case anyone wants to refer to what the option did in 1.0. It's kind of unfortunate that would also expose it publicly though. Maybe there's a way to merge in type declarations that are only used for generating the docs

Reply:
We could always change it to `trackServerFetches: never` and mark it deprecated.

Reply:
That probably works well enough